libmodernize/main.py

```diff
@@ -27,6 +27,8 @@ def format_usage(usage):
     """Method that doesn't output "Usage:" prefix"""
     return usage
 
+_lib2to3_has_output_dir = hasattr(StdoutRefactoringTool([], {}, False, False, False), '_output_dir')
```

Comment:
I think we can drop this check - we don't aim to support the older Pythons that don't have this.